### PR TITLE
Use WordPressAuthenticator v7.0.0-beta.1 to drop Google SignIn SDK

### PR DESCRIPTION
**Pull request by mokagio**
See changes from https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/777 and https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/778

I also tested this manually by running a full sign in flow in the Jetpack installable build.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift [hunk 1]
```diff
@@ -48,8 +48,8 @@ extension LoginEpilogueUserInfo {
 
     /// Updates the Epilogue properties, given a SocialService instance.
     ///
```
Comment (mokagio) [inline WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift]: I noticed this comment being outdated as it mentions the `SocialService` which is now `SocialUser`.

Thinking about it, I don't think it adds any information that a reader wouldn't already gather from the method signature.

```suggestion
```

**Comment by crazytonyli** [comment]
@mokagio FYI, there are some conflicts in this PR.

**Comment by mokagio** [comment]
> @mokagio FYI, there are some conflicts in this PR.

Thank you for reporting. We move too fast! 😭 

Just kidding. But it is annoying when this happens...

**Review by crazytonyli — APPROVED** [review]
🎉

**Pull request merged**